pennylane/operation.py

```diff
@@ -1213,7 +1213,7 @@ def __add__(self, other):
             return qml.ops.Sum(  # pylint: disable=no-member
                 self,
                 qml.ops.SProd(  # pylint: disable=no-member
-                    scalar=other, base=qml.Identity(wires=self.wires)
+                    scalar=other, base=qml.Identity(wires=self.wires[0])
                 ),
             )
```

Comment:
Not sure how we missed this when reviewing the other PR, but this might be problematic, especially when you have a multi wire operator summed with a number

Reply:
Oh yeah, good point.  Really wish we had a multi-qubit Identity right about now. Maybe we should revert and not support adding numbers to operators just yet?

Reply:
😆, multi-qubit identity is now higher on the priority list haha. We discussed a quick work around, so this shouldn't be a big issue

Reply:
We use a `Prod` of `Identity` ops for each different wire for now.